tests/unit/test_ip.py

```diff
@@ -637,6 +665,16 @@ def test_cidr_to_netmask_fail():
         ip.cidr_to_netmask(**data)
 
 
+@pytest.mark.parametrize("data", NETMASK_WILDCARDMASK)
+def test_netmask_to_wildcardmask(data):
+    assert ip.netmask_to_wildcardmask(**data["sent"]) == data["received"]
+
+
+@pytest.mark.parametrize("data", WILDCARDMASK_NETMASK)
+def test_wildcardmask_to_wildcardmask(data):
+    assert ip.wildcardmask_to_netmask(**data["sent"]) == data["received"]
+
```

Comment:
Can you add some invalid inputs and likely add such validation on the function? 

Especially for `wildcardmask_to_netmask`, as we consider things that do not have an equivalent like `0.0.255.0`.